src/test/resources/ping-pong/daml.yaml

```diff
@@ -3,13 +3,9 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-sdk-version: 1.15.0
+sdk-version: 2.2.0
 name: ping-pong
 source: daml
-init-script: Main:init
```

Comment:
Is it not a problem that you remove the init script?
I think it is called from the library "manually" anyway, correct?

Reply:
checked it. This init-script doesn't run on the sandbox start. You have to specify script inside builder method
 .moduleAndScript("Main", "init") in order to run it